docs/developers-guide.rst

```diff
@@ -0,0 +1,209 @@
+5. Classes should either use direct variable access, or Python’s property mechanism for setting
+object instance variables.
+6. Classes should use the builtin `super() <https://docs.python.org/3/library/functions.html#super>`_ function
+when making calls to methods in their super-class(es), unless there are specific reasons not to. Also,
+`super() <https://docs.python.org/3/library/functions.html#super>`_ should be used consistently in all subclasses,
+since it does not work otherwise.
+7. Multiple inheritance should be avoided in general without good justification.
+8. The ``__init__.py`` files for each module should not contain any significant implementation code.
+Each ``__init__.py`` can contain docstrings and code for organizing the module layout.
+
+Documentation and Testing
+^^^^^^^^^^^^^^^^^^^^^^^^^
+1. American English is the default language for all documentation strings and inline commands.
+Variables names should also be based on English words as far as possible.
+2. Documentation strings must be present for all public classes/methods/functions. Furthermore, inclusion of
+examples or tutorials in the package documentation is strongly recommended and encouraged.
+3. Write usage examples in the docstrings of all classes and functions whenever possible.
+These examples should be short and simple to reproduce, so that users would be able to copy them verbatim
+and run them. These examples should also, whenever possible, be in the
+`doctest <https://docs.astropy.org/en/stable/development/testguide.html#doctests>`_
+format and will be executed as part of the test suite.
+4. Unit tests should be provided for as many public methods and functions as possible.
+
+Glue-solar Documentation Rules
+------------------------------
+We recommend following for example the
+`SunPy Documentation Rules <https://docs.sunpy.org/en/latest/dev_guide/documentation.html>`_, though it is not
+required to follow the rules strictly as we prefer to use these as guidelines rather than rules.
+
+Overview
+^^^^^^^^
+For the RST files, we recommend a one-sentence-per-line rule and ignore the line length.
+
+Sphinx
+^^^^^^
+All of the ``Glue-solar`` documentation (like this page) is built with
+`Sphinx <https://www.sphinx-doc.org/en/stable/>`_, which is a tool very suitable for documenting Python projects.
+Sphinx works by parsing files written in a
+`Media-wiki-like syntax <http://docutils.sourceforge.net/docs/user/rst/quickstart.html>`_ called
+`reStructuredText <http://docutils.sourceforge.net/rst.html>`_. In addition to parsing static files
+of reStructuredText, Sphinx can be instructed to parse code comments as well. In fact, in addition
+to what you are reading right now, the `Python documentation <https://www.python.org/doc/>`_
+has also created using Sphinx.
+
+All of the ``Glue-solar`` documentation is contained in the “docs” folder and code documentation strings.
+
+To build the docs with tox, in the root directory run::
+
+    tox -e build_docs
```

Comment:
Is tox setup for glue-solar?

Reply:
Yes, I can do `tox -e build_docs` locally at the time being